proxy/grpc/generate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+# go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+# go install github.com/fatih/gomodifytags@v1.13.0
```

Comment:
No need to replace tags, this is only for JSON

Reply:
fixed